pkg/mount/mount.go

```diff
@@ -1,3 +1,4 @@
+//go:build linux
```

Comment:
is this intended? What does it do?

Reply:
It was added automatically by VSCode.
those serve the same purpose as `// +build linux`, but for go version 1.6+. 
Those are build tags. Basically saying the build should be under linux environment. 
Doc: [constraints are satisfied](https://pkg.go.dev/cmd/go#hdr-Build_constraints)